clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/CreateServerGroupAtomicOperation.java

```diff
@@ -334,18 +334,42 @@ protected RegisterTaskDefinitionRequest makeTaskDefinitionRequest(
     return request;
   }
 
-  protected RegisterTaskDefinitionRequest makeTaskDefinitionRequestFromArtifact(
-      String ecsServiceRole, EcsServerGroupName newServerGroupName) {
+  private RegisterTaskDefinitionRequest getSpelProcessedArtifact() {
+    RegisterTaskDefinitionRequest requestTemplate = null;
+    for (Map<Object, Object> serializedSpelProcessedTaskDefArtifact :
+        description.getSpelProcessedTaskDefinitionArtifact()) {
+      if (serializedSpelProcessedTaskDefArtifact != null) {
+        requestTemplate =
+            mapper.convertValue(
```

Comment:
I think we can just return this line, no need for the separate `requestTemplate` variable

Reply:
Make sense. Updated.